dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -840,23 +903,135 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 			return;
 		}
 
+		int count = 1;
+
+		if (args.size() == 3) {
+			if (!GeneralUtils::TryParse(args[2], count))
+			{
+				ChatPackets::SendSystemMessage(sysAddr, u"Invalid item count.");
+				return;
+			}
+
+			count = std::stoi(args[2]);
```

Comment:
Why do you parse the string here again? `count` should already contain the parsed value from the `TryParse` call above, shouldn't it? Same applies to the "mailitemall" command below.
(I am just curious.)

Reply:
Good catch, @Verathian did you do this on purpose?